src/main/java/com/rakovets/course/java/core/practice/exception_handling/Task1.java

```diff
@@ -0,0 +1,13 @@
+package com.rakovets.course.java.core.practice.exception_handling;
+
+public class Task1 {
+    public static void main(String[] args) {
+        String string = null;
+
+        try {
+           int x = string.length();
+        } catch (Exception nullEx) {
```

Comment:
You shouldn't use a high level of abstraction for this exception.
```suggestion
        } catch (NullPointerException nullEx) {
```